specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -900,22 +1158,66 @@
         },
         "targetIqn": {
           "type": "string",
-          "description": "iSCSI target iqn (iSCSI Qualified Name); example: iqn.2005-03.org.iscsi:server"
+          "description": "iSCSI target IQN (iSCSI Qualified Name); example: \"iqn.2005-03.org.iscsi:server\"."
+        }
+      }
+    },
+    "IscsiTargetCreateProperties": {
+      "type": "object",
+      "description": "Properties for iSCSI Target create or update request.",
+      "required": [
+        "tpgs",
+        "targetIqn"
+      ],
+      "properties": {
+        "tpgs": {
+          "type": "array",
+          "description": "List of iSCSI target portal groups.",
+          "items": {
+            "$ref": "#/definitions/TargetPortalGroupCreate"
+          },
+          "maxLength": 1,
```

Comment:
Just a list of 1? Make sure to let the user know by adding the constraint in the description.